### PR TITLE
[ci-visibility] Add `test.toolchain` containing the package manager name and version

### PR DESCRIPTION
**Pull request by juan-fernandez**
### What does this PR do?
Add `test.toolchain` containing the package manager name and version

### Motivation
Provide more information to the user about their test sessions.

### CODE REVIEW

File: packages/datadog-plugin-jest/src/index.js [hunk 1]
```diff
@@ -42,9 +40,6 @@ class JestPlugin extends CiPlugin {
       process.on('message', handler)
     }
 
-    this.testEnvironmentMetadata = getTestEnvironmentMetadata('jest', this.config)
-    this.codeOwnersEntries = getCodeOwnersFileEntries()
```
Comment (juan-fernandez) [inline packages/datadog-plugin-jest/src/index.js:46]: unrelated fix: these values are already calculated at `CiPlugin`

**Review by rochdev — APPROVED** [review]
